src/screens/Classes.js

```diff
@@ -19,6 +20,7 @@ class Classes extends Component {
     return (
       <ScreenContainer screenTitle='Classes'>
         <TextBody bodyColor={theme.COLOR_BLACK}>Classes!</TextBody>
+        <FeatureButtons></FeatureButtons>
```

Comment:
@stellakimnguyen, this was added just for testing, we can remove it!

Reply:
Yes, we can remove!!